packages/types/src/runtime/transaction-payment/TransactionPayment.ts

```diff
@@ -0,0 +1,15 @@
+import {AnyAssetId} from '@cennznet/types/types';
```

Comment:
it is still in `types` package, but it is a kind a util function (and cennznet/utils seems not needed)... 
What if just place it in the file of `packages/api/src/derives/fees/estimateFee.ts`?

Otherwise, potentially new onboard developers would add more similar utils functions into `types` package? Then the package gonna be out of control and we potentially need to make another round refactoring... :)

Reply:
@holygits ?

Reply:
Except this point, it looks good to me. 

Any inputs on what kind of functions we should consider to put inside `types` package? @holygits

Reply:
The constructor should go with the type it is constructing so `@types` seems good place